.github/workflows/jira.pr.yml

```diff
@@ -1,20 +0,0 @@
-on: 
```

Comment:
This was never enabled, need to remove before enabling github actions on this repo.